activestorage/app/controllers/concerns/active_storage/streaming.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module ActiveStorage::Streaming
+  DEFAULT_BLOB_STREAMING_DISPOSITION = "inline"
+
+  include ActionController::Live
```

Comment:
is this module included in `api_only` mode? Asking because otherwise they won't be able to use `ActiveStorage` anymore

Reply:
api mode does not support streaming? That sounds like something we should fix.

Reply:
it was a legit question, I don't really know 😇 maybe everything is fine

Reply:
Please do verify! It's a good question.

Reply:
After checking https://github.com/rails/rails/issues/32208 and https://github.com/rails/rails/pull/32238 seems that the issue with `ActionController::Base` is only present on direct uploads because of some missing middlewares for `api_only` apps. So `ActionController::Live` won't cause any issues.